offline/docs.md

```diff
@@ -324,26 +324,30 @@ For full docs with details and explanations please see https://github.com/wireap
 First, make sure you have a certificate for `sftd.<yourdomain>`. This could be the same wildcard or SAN certificate
 you used at previous steps.
 
-If you want to restrict SFT to certain nodes, make sure that in your inventory
-you have annotated all the nodes that are able to run sftd workloads correctly.
+Next, copy the values/sftd/prod-example-values.yaml to values/sftd/values.yaml, and change the contents accordingly. 
+
+ * If your turn servers can be reached on their public IP by the SFT service, Wire recommends you enable cooperation between TURN and SFT. add a line reading `turnDiscoveryEnabled: true` to your values file.
+
+#### Deploying
+If you want to restrict SFT to certain nodes, make sure that in your inventory file you have annotated all of the nodes that are able to run sftd workloads with a node label indicating they are to be used, and their external IP, if they are behind a 1:1 firewall (Wire recommends this.).
 ```
 kubenode3 node_labels="{'wire.com/role': 'sftd'}" node_annotations="{'wire.com/external-ip': 'XXXX'}"
 ```
 
-If these weren't already set; you should rerun :
+If these values weren't already set earlier in the process you should rerun ansible to set them:
 ```
 d ansible-playbook -i ./ansible/inventory/offline ansible/kubernetes.yml --skip-tags bootstrap-os,preinstall,container-engine
 ```
 
-
-If you are restricting SFT to certain nodes, use `nodeSelector` to run on specific nodes (of course **replace the domains with yours**):
+If you are restricting SFT to certain nodes, use `nodeSelector` to run on specific nodes (and of course **replace the example.com domains with yours**):
```

Comment:
I think you can remove `and of course`. These three words don't add much... :nerd_face: